### ISSUE TITLE
May you provide training logs for pretrain model?

### ISSUE THREAD

**Issue by Sunting78** [amirbar/DETReg#47]
Thanks for the great work! I want to train the pretrain model based on IMN1k. The class number is 92, but the class label is 1 only. The loss of cate_error is always 0 or 100. May you provide training logs for pretrain model?

**Comment by amirbar** [comment]
Right, please see - #14
